harness-delegate-ng/templates/_helpers.tpl

```diff
@@ -68,6 +68,18 @@ Fetch access level using kubernetes permission value
   {{- end }}
 {{- end }}
 
+
+{{/*
+Check if custom role is provided in k8sPermissionsType
+*/}}
+{{- define "harness-delegate-ng.useCustomRole" -}}
```

Comment:
if user manually set useCustomRole in value.yaml file, which one takes precedence ? 
a buggy setting could be 
useCustomRole: true
k8sPermissionsType: CLUSTER_ADMIN

Reply:
There's is no setting for useCustomRole: true. If something else is mentioned apart from CLUSTER_ADMIN, CLUSTER_VIEWER or NAMESPACE_ADMIN. We will treat it like its a custom role.